File: 백준/Bronze/25494. 단순한 문제 （Small）/단순한 문제 （Small）.c
#include <stdio.h>
int main(void) {
    int T, a, b, c, cnt;
    scanf("%d", &T);
    for (int i = 0; i < T; i++) {
        scanf("%d %d %d", &a, &b, &c);
        cnt = 0;
        for (int x = 0; x < a; x++) {
            for (int y = 0; y < b; y++) {
                for (int z = 0; z < c; z++) {
                    if ((x + 1) % (y + 1) == (y + 1) % (z + 1) && (y + 1) % (z + 1) == (z + 1) % (x + 1))
                        cnt += 1;
                }
            }
        }
        printf("%d\n", cnt);
    }
    return 0;
}

File: 백준/Bronze/25494. 단순한 문제 （Small）/단순한 문제 （Small）.py
import sys
input=lambda:sys.stdin.readline().rstrip()
T=int(input())
for _ in range(T):
    a,b,c=map(int,input().split())
    cnt=0
    for x in range(a):
        for y in range(b):
            for z in range(c):
                if (x+1)%(y+1)==(y+1)%(z+1)==(z+1)%(x+1):
                    cnt+=1
    print(cnt)